### ISSUE TITLE
Terraform outputs from some ComponentDefinitions are not generated

### ISSUE THREAD

**Issue by zzxwill** [oam-dev/kubevela#3224]
**Describe the bug**

When using `vela def doc-gen` to [generate documentation for a Terraform cloud resource](http://kubevela.net/docs/platform-engineers/components/component-terraform#generate-documentation), the outputs for Terraform configuration should be generated as below:

![image](https://user-images.githubusercontent.com/2805315/153376472-2fa5f3a4-13bb-4485-9650-2bfa7feaccb7.png)
http://kubevela.net/docs/end-user/components/cloud-services/terraform/alibaba-ack#outputs

But some cloud resources, like [aws-iam-account](http://kubevela.net/docs/end-user/components/cloud-services/terraform/aws-iam-account) wan't.

**To Reproduce**

**Expected behavior**

**Screenshots**

**KubeVela Version**

**Cluster information**

**Additional context**

**Comment by Vrukshali-26** [comment]
Can I work on this issue? @zzxwill

**Comment by zzxwill** [comment]
@Vrukshali-26 Yes, please.

**Comment by Vrukshali-26** [comment]
Hey @zzxwill Can I get some resources to understand the entire Kubevela Architecture?

**Comment by zzxwill** [comment]
@Vrukshali-26 Yes, you can look through the resources https://kubevela.io/docs/getting-started/architecture.

**Comment by briankane** [comment]
This issue has been inactive for over 2 years and is scheduled for closure as part of our backlog maintenance.

  **If this issue is still relevant:**
  - Please comment below to keep it open
  - Provide updated context if the original description is outdated

  **Timeline:** If there's no activity by **Friday, October 24th**, this issue will be automatically closed.

  **Note:** Issues can always be reopened if they become relevant again. We encourage the community to reopen and reprioritize any incorrectly 
  closed issues.

**Comment by briankane** [comment]
This issue has been closed following the stale issue notice. 

If this issue is still relevant and should remain open, please:
1. Reopen the issue
2. Remove the 'stale' label
3. Provide an update on the current status or importance

Thank you for your understanding and contribution to KubeVela!
